src/services/messages-processor/service.ts

```diff
@@ -235,30 +238,33 @@ export const makeMessagesProcessor = ({
     eventsNumber: number
     messages: ExitMessage[]
   }) => {
-    logger.info('Job started')
+    logger.info('Job started', { OPERATOR_ID: config.OPERATOR_ID })
```

Comment:
maybe `OPERATOR_ID` -> `operatorId`

need to add module id as well

Reply:
Moved this comment to the original PR #36